
Showing logs for build [32mseldonio-seldon-core-pr-1424-li-86776-6[0m stage [32mpr-build-comment[0m and container [32mstep-credential-initializer-7wxq5[0m
{"level":"warn","ts":1581430343.8059633,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1581430343.8064048,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1424-li-86776-6[0m stage [32mpr-build-comment[0m and container [32mstep-working-dir-initializer-28vwc[0m
{"level":"warn","ts":1581430344.8805702,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581430344.8822963,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1424-li-86776-6[0m stage [32mpr-build-comment[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1424-li-86776-6[0m stage [32mpr-build-comment[0m and container [32mstep-create-dir-workspace-ld4qd[0m
{"level":"warn","ts":1581430351.0308754,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581430351.0326087,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1424-li-86776-6[0m stage [32mpr-build-comment[0m and container [32mstep-git-source-seldonio-seldon-core-pr-1424-li-l5529[0m
{"level":"warn","ts":1581430351.3221161,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1581430353.782713,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/SeldonIO/seldon-core.git @ master in path /workspace/source"}

Showing logs for build [32mseldonio-seldon-core-pr-1424-li-86776-6[0m stage [32mpr-build-comment[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:9e81a0067ca9fd7a18a41b32e37e5e0e68cb076c,1424:556f4ef11b4ae0fc2874b0e4c7d9bbbb3280bba0
DEBUG: ran git fetch --unshallow origin 556f4ef11b4ae0fc2874b0e4c7d9bbbb3280bba0: 9e81a0067ca9fd7a18a41b32e37e5e0e68cb076c: in 
DEBUG: ran git checkout master in 
DEBUG: ran git reset --hard 9e81a0067ca9fd7a18a41b32e37e5e0e68cb076c in 
DEBUG: ran clean --force -d . in 
DEBUG: ran git merge 556f4ef11b4ae0fc2874b0e4c7d9bbbb3280bba0 in 
Merged SHA 556f4ef11b4ae0fc2874b0e4c7d9bbbb3280bba0 with commit message 'run linter' into base branch master
Merged SHA 412215bc4b8b429bca93f986f6cb994189744bf4 with commit message 'Remove prints from python/tests. Closes #1372' into base branch master

Showing logs for build [32mseldonio-seldon-core-pr-1424-li-86776-6[0m stage [32mpr-build-comment[0m and container [32mstep-step2[0m
+++ dirname ./add-pr-build-comment
++ cd .
++ pwd
+ STARTUP_DIR=/workspace/source/ci
+ REPO_OWNER=SeldonIO
+ REPO_NAME=seldon-core
+ PULL_NUMBER=1424
+ BUILD_NUMBER=6
+ PIPELINE_CONTEXT=lint
++ cat
+++ date
+ COMMENT_MSG='Tue Feb 11 14:12:39 UTC 2020
The logs for [lint] [6] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1424/6.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1424 --build=6'
+ ** step pr comment --owner=SeldonIO --repository=seldon-core --pull-request=1424 '--comment=Tue Feb 11 14:12:39 UTC 2020
The logs for [lint] [6] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1424/6.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1424 --build=6'

Showing logs for build [32mseldonio-seldon-core-pr-1424-li-86776-6[0m stage [32mpr-build-comment[0m and container [32mstep-source-mkdir-seldonio-seldon-core-pr-1424-li-4wmx6[0m
{"level":"warn","ts":1581430362.185875,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581430362.189212,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1424-li-86776-6[0m stage [32mpr-build-comment[0m and container [32mstep-source-copy-seldonio-seldon-core-pr-1424-li-gq6m6[0m
{"level":"warn","ts":1581430362.4768143,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581430362.7135246,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r source/. /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1424-li-86776-6[0m stage [32mpython-lint-task[0m and container [32mstep-credential-initializer-n9fbr[0m
{"level":"warn","ts":1581430381.178987,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1581430381.1796403,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1424-li-86776-6[0m stage [32mpython-lint-task[0m and container [32mstep-working-dir-initializer-9cl2g[0m
{"level":"warn","ts":1581430382.1715245,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581430382.1732335,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1424-li-86776-6[0m stage [32mpython-lint-task[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1424-li-86776-6[0m stage [32mpython-lint-task[0m and container [32mstep-create-dir-workspace-pwwgf[0m
{"level":"warn","ts":1581430386.2487962,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581430386.250131,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1424-li-86776-6[0m stage [32mpython-lint-task[0m and container [32mstep-source-copy-workspace-brgjf[0m
{"level":"warn","ts":1581430386.47469,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581430388.3597577,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r /pvc/pr-build-comment/workspace/. /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1424-li-86776-6[0m stage [32mpython-lint-task[0m and container [32mstep-python-lint[0m
make: Entering directory '/workspace/source/python'
pip install black==19.3b0
Collecting black==19.3b0
  Downloading https://files.pythonhosted.org/packages/30/62/cf549544a5fe990bbaeca21e9c419501b2de7a701ab0afb377bc81676600/black-19.3b0-py36-none-any.whl (89kB)
Collecting toml>=0.9.4
  Downloading https://files.pythonhosted.org/packages/a2/12/ced7105d2de62fa7c8fb5fce92cc4ce66b57c95fb875e9318dba7f8c5db0/toml-0.10.0-py2.py3-none-any.whl
Collecting click>=6.5
  Downloading https://files.pythonhosted.org/packages/fa/37/45185cb5abbc30d7257104c434fe0b07e5a195a6847506c074527aa599ec/Click-7.0-py2.py3-none-any.whl (81kB)
Collecting attrs>=18.1.0
  Downloading https://files.pythonhosted.org/packages/a2/db/4313ab3be961f7a763066401fb77f7748373b6094076ae2bda2806988af6/attrs-19.3.0-py2.py3-none-any.whl
Collecting appdirs
  Downloading https://files.pythonhosted.org/packages/56/eb/810e700ed1349edde4cbdc1b2a21e28cdf115f9faf263f6bbf8447c1abf3/appdirs-1.4.3-py2.py3-none-any.whl
Installing collected packages: toml, click, attrs, appdirs, black
Successfully installed appdirs-1.4.3 attrs-19.3.0 black-19.3b0 click-7.0 toml-0.10.0
WARNING: You are using pip version 19.3.1; however, version 20.0.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
black \
	--check ./ ../testing \
	--exclude "(testing/scripts/proto|seldon_core/proto/|.eggs|.tox)"
All done! ‚ú® üç∞ ‚ú®
54 files would be left unchanged.
make: Leaving directory '/workspace/source/python'
